serverFiles/models/api.models.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs/promises');
+// const endpoints = require('../endpoints.json');
+
```

Comment:
we don't need a model for this, so we can pop this functionality in the controller :)